src/components/Button.svelte

```diff
@@ -0,0 +1,148 @@
+<script lang="ts">
+
+  import type { Snippet } from "svelte";
+  import { phosphorIcon } from "@data/icons";
+  import Icon from "@iconify/svelte";
+
+
+  interface Props {
+    children: Snippet;
+    style?: 'mauve' | 'peach' | 'blue' | 'green' | 'transparent';
+    small?: boolean;
```

Comment:
Instead of taking in `small`, it'll be better to take in an explicit `width` and `height`. We could default those to some number if we want.

Reply:
`small` applies to the button size in general, not just the icon size. I don't think we want to use fixed dimensions for buttons?

Reply:
Is `small` not referring to the icon within the button?